website/docs/concepts/step-function/index.md

```diff
@@ -3,9 +3,21 @@
 AWS's Step Function service offers a powerful primitive for building long-running and short-running state machine workflows in the cloud without managing complex infrastructure.
 
 ```ts
-new StepFunction(scope, "F", async (payload) => {
+new StepFunction(scope, "F", async () => {
+  return "hello, world";
+});
+```
+
+The Step Function can make calls to other resource like a Lambda [Function](../function/), handle a `payload`, introduce wait time, add conditional logic, throw errors, and more.
+
+```ts
+const serviceCall = new Function(scope, "serviceCall", async (input: string) => {
+  console.log(input);
+});
+new StepFunction(scope, "F", async (payload: { property?: string }) => {
   if (payload.property) {
-    await serviceCall(payload);
+    SFN.waitFor(10);
+    await serviceCall(payload.property);
```

Comment:
Maybe add a comment here emphasizing how the `serviceCall` Function Construct can be called like an ordinary function

Reply:
> // the lambda function `serviceCall` is invoked like an ordinary function from the state machine.